components/Trade/Advanced/RightPanel/InsuranceInfo/index.tsx

```diff
@@ -8,13 +8,15 @@ import TooltipSelector from '@components/Tooltips/TooltipSelector';
 
 export default styled(({ className }) => {
     const { poolInfo } = useContext(InsuranceContext);
-    const poolHealth = poolInfo?.health ?? 0;
+    const poolHealth = poolInfo?.health ?? defaults.health;
+    const poolLiquidity = poolInfo?.liquidity ?? defaults.liquidity;
+    const poolTarget = poolInfo?.target ?? defaults.target;
     return (
         <div className={className}>
             <h3>
                 <TooltipSelector tooltip={{ key: 'insurance-pool-health' }}>Insurance Pool Health</TooltipSelector>
             </h3>
-            <PoolHealth health={parseFloat(poolHealth.toFixed(2)) ?? defaults.health.toNumber()} />
+            <PoolHealth health={parseFloat(poolHealth.toFixed(2)) ?? defaults.health} />
```

Comment:
I think this parses the value twice since pool health is progress bar

Reply:
Removed PoolHealth, just using ProgressBar